pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "cloupe"
+dynamic = ["version"]
```

Comment:
dynamic property needs 
```toml
[tool.hatch.version]
path = "src/cloupe/__init__.py"
```

as described in the docs: https://hatch.pypa.io/1.13/version/

Reply:
Done